includes/payment-methods/class-cc-payment-method.php

```diff
@@ -70,6 +70,6 @@ public function get_title( string $account_country = null, $payment_details = fa
 	 * @return string
 	 */
 	public function get_testing_instructions() {
-		return __( '<strong>Test mode:</strong> use the test VISA card 4242424242424242 with any expiry date and CVC. Other payment methods may redirect to a Stripe test page to authorize payment. More test card numbers are listed <a>here</a>.', 'woocommerce-payments' );
+		return __( 'Use test card 4242 4242 4242 4242 <span></span> or refer to our <a>Testing Guide</a>.', 'woocommerce-payments' );
```

Comment:
```suggestion
		return __( 'Use test card 4242 4242 4242 4242 <span></span> or refer to our <a>testing guide</a>.', 'woocommerce-payments' );
```

(honest question) does "testing guide" need to be capitalized? I personally find it odd to suddenly see the link capitalized - maybe that's just me :D 
Please do let me know!

Reply:
I felt the same seeing it and I'm the one who proposed it. Switch to lower case.